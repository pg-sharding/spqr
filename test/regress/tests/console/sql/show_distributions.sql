CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
CREATE DISTRIBUTION ds2 COLUMN TYPES varchar, integer;

SHOW distributions;

DROP DISTRIBUTION ALL CASCADE;
DROP KEY RANGE ALL;
