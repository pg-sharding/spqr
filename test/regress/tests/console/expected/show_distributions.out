
		SPQR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
    add distribution    
------------------------
 distribution id -> ds1
(1 row)

CREATE DISTRIBUTION ds2 COLUMN TYPES varchar, integer;
    add distribution    
------------------------
 distribution id -> ds2
(1 row)

SHOW distributions;
 Distribution ID |  Column types   
-----------------+-----------------
 ds1             | integer
 ds2             | varchar,integer
(2 rows)

DROP DISTRIBUTION ALL CASCADE;
   drop distribution    
------------------------
 distribution id -> ds1
 distribution id -> ds2
(2 rows)

DROP KEY RANGE ALL;
 drop key range 
----------------
(0 rows)

